tests/Microsoft.DotNet.Framework.Docker.Tests/projects/dotnetapp-4.8/Dockerfile

```diff
@@ -3,6 +3,7 @@ ARG BASE_RUNTIME_IMAGE
 
 
 FROM $BASE_BUILD_IMAGE as builder
+
```

Comment:
Consider reverting to keep the format of the test Dockerfiles consistent across versions.